subprojects/docs/src/samples/userguide/tutorial/logging/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,39 @@
+// tag::use-slf4j[]
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+val slf4jLogger = LoggerFactory.getLogger("some-logger")
```

Comment:
:o: nit pic, the corresponding Groovy sample uses the following declaration: `Logger slf4jLogger = ...` which makes the `org.slf4j.Logger` import necessary. It could be changed to `def slf4jLogger = ...` to make it looks more similar to the Kotlin sample.